src/managers/GuildApplicationCommandManager.js

```diff
@@ -119,6 +120,87 @@ class GuildApplicationCommandManager extends ApplicationCommandManager {
     );
   }
 
+  /**
+   * Add permissions to a command.
+   * @param {ApplicationCommandResolvable} command The command to edit the permissions for
+   * @param {ApplicationCommandPermissionData[]} permissions The permissions to add to this command
+   * @returns {Promise<ApplicationCommandPermissions[]>}
+   * @example
+   * // Block the @everyone role from command permissions
+   * guild.commands.addPermissions('123456789012345678', [
+   *   {
+   *     id: guild.roles.everyone.id,
+   *     type: 'ROLE',
+   *     permission: false
+   *   },
+   * ])
+   *   .then(console.log)
+   *   .catch(console.error);
+   */
+  async addPermissions(command, permissions) {
+    const id = this.resolveID(command);
+    if (!id) throw new TypeError('INVALID_TYPE', 'command', 'ApplicationCommandResolvable');
+
+    const perms = await this.fetchPermissions(id).catch(() => []);
+    for (const perm of perms) {
+      if (!permissions.find(x => x.id === perm.id)) {
+        permissions.push(perm);
+      }
+    }
+
+    const data = await this.commandPath(id).permissions.put({
+      data: { permissions: permissions.map(perm => this.constructor.transformPermissions(perm)) },
+    });
+
+    return data.permissions.map(perm => this.constructor.transformPermissions(perm, true));
+  }
```

Comment:
```suggestion
    return this.setPermissions(id, permissions);
```
Just call the existing method from here after calculating the new permission set, no need to duplicate code.